tests/RestApiSuite/TestPost/content.txt

```diff
@@ -1,12 +1,20 @@
 !|post|${URL}/delete/users|
 
-!|post    |${URL}/users|
-|FirstName|LastName    |
-|Tony     |Soprano     |
-|Jennifer |Melfi       |
-
-!|Query:http result as table|${URL}/users|
-|FirstName                  |LastName    |
-|Tony                       |Soprano     |
-|Jennifer                   |Melfi       |
+!|post    |${URL}/users    |
+|FirstName|LastName|IsMafia|
+|Tony     |Soprano |True   |
+|Jennifer |Melfi   |False  |
+
+!|script|RestTools                       |
+|check|get_json_str|${URL}/users|[{"FirstName": "Tony", "IsMafia": true, "LastName": "Soprano", "id": 1}, {"FirstName": "Jennifer", "IsMafia": false, "LastName": "Melfi", "id": 2}]|
```

Comment:
@AntonTimiskov - there is a change in existing behavior of POST fixture - please check whether you expect issues with this in DMP test suite